.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "svg.preview.background": "editor"
+}
```

Comment:
Let's remove it. Thank you.

Reply:
BTW, here is how to setup a global ignore file for your Git.

https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
Ok, thanks!